packages/graphql-yoga/__tests__/integration.spec.ts

```diff
@@ -241,3 +242,190 @@ describe('Incremental Delivery', () => {
     expect(getCounterValue()).toBe(counterValue2)
   })
 })
+
+describe('HTTP framework integration', () => {
+  describe('fastify', () => {
+    const app = buildFastifyApp()
+
+    beforeAll(async () => {
+      await app.listen(4411)
+    })
+
+    afterAll(async () => {
+      await app.close()
+    })
+
+    it('renders GraphiQL', async () => {
+      const response = await request(app.server).get('/graphql').set({
+        accept: 'text/html',
+      })
+
+      expect(response.statusCode).toEqual(200)
+      expect(response.text).toContain('<title>GraphiQL</title>')
+    })
+
+    it('handles query operation via POST', async () => {
+      const response = await request(app.server)
+        .post('/graphql')
+        .set({ 'content-type': 'application/json' })
+        .send(
+          JSON.stringify({
+            query: /* GraphQL */ `
+              query {
+                __typename
+              }
+            `,
+          }),
+        )
+
+      expect(response.statusCode).toEqual(200)
+      expect(response.body).toStrictEqual({
+        data: {
+          __typename: 'Query',
+        },
+      })
+    })
+
+    it('handles query operation via GET', async () => {
+      const response = await request(app.server)
+        .get('/graphql')
+        .query({
+          query: /* GraphQL */ `
+            query {
+              __typename
+            }
+          `,
+        })
+
+      expect(response.statusCode).toEqual(200)
+      expect(response.body).toStrictEqual({
+        data: {
+          __typename: 'Query',
+        },
+      })
+    })
+
+    it('handles mutation operation via POST', async () => {
+      const response = await request(app.server)
+        .post('/graphql')
+        .set({ 'content-type': 'application/json' })
+        .send(
+          JSON.stringify({
+            query: /* GraphQL */ `
+              mutation {
+                __typename
+              }
+            `,
+          }),
+        )
+
+      expect(response.statusCode).toEqual(200)
+      expect(response.body).toStrictEqual({
+        data: {
+          __typename: 'Mutation',
+        },
+      })
+    })
+
+    it.skip('rejects mutation operation via GET with an useful error message', async () => {
```

Comment:
skipped on purpose?

Reply:
skipped on purpose as it is not yet implemented :D